selectable APY on homepage

rolandpo opened the pull request:
This adds the option to choose between APY sample periods on the homepage. Options are 7, 30, 60, 90, 365 days, similar to what has already been implemented on the swap page, but can cut these down. Default is set to 30 days
also localStorage is shared between the selection on the homepage and swap page

issue #1026

<img width="659" alt="Screenshot 2022-07-08 at 18 18 58" src="https://user-images.githubusercontent.com/62400812/178031319-6e54c373-b9d0-4ab0-9042-fb588d815d5e.png">

<img width="628" alt="Screenshot 2022-07-08 at 18 19 21" src="https://user-images.githubusercontent.com/62400812/178031374-4c460e84-43ba-4334-8a0a-4d70673b02ce.png">

micahalcorn commented:
@rolandpo I actually meant for this to be applied to the text under the big number below the fold. Do your changes address that section too or just the little banner in the nav? It probably belongs in both places but let's be sure to include it where we use the words and have the big call-to-action.

<img width="185" alt="Screen Shot 2022-07-08 at 12 44 40 PM" src="https://user-images.githubusercontent.com/273937/178060182-d6cb296f-ad95-4b6a-a61f-42ac2078acb1.png">

rolandpo commented:
removed the apy selection from the top (and set it to 30 days) and added it down below

<img width="634" alt="Screenshot 2022-07-12 at 20 17 56" src="https://user-images.githubusercontent.com/62400812/178565826-7bff0744-da2b-4f96-bcf1-7c7567c24e47.png">

<img width="651" alt="Screenshot 2022-07-12 at 20 18 06" src="https://user-images.githubusercontent.com/62400812/178565867-7601ed8c-bdbb-4736-812c-cc2e450bf3ba.png">
